packages/fhir-converter/src/templates/cda/DataType/Coding.hbs

```diff
@@ -3,6 +3,6 @@
 	"display": "{{{ parseReferenceData code.displayName }}}",
 	"version":"{{code.codeSystemVersion}}",
 	{{#with (evaluate 'ValueSet/SystemReference.hbs' code=code.codeSystem) as |system|}}
-		"system": "{{system.oid}}"",
+		"system": "{{system.oid}}",
```

Comment:
this seems important enough to have included in a patch to master?

Reply:
I should have in retrospect, but for now this is getting shipped soon enough that it should be ok to just bundle it along with here